File: Golang/beecrowd-challenges/bee-1011.go
package main

import (
	"fmt"
	"math"
)

func main(){
	pi := 3.14159
	var r float64; fmt.Scanln(&r)

	volume := (4.0/3.0) * pi * math.Pow(r, 3) 

	fmt.Printf("VOLUME = %.3f\n", volume)
}

File: javascript/javascript/JavaScriptHackerRank/beecrowd/bee-1000-1037/bee-1011.js
const input = require('fs').readFileSync('./dev/stdin', 'utf8');
const lines = input.split('\n');
const PI = 3.14159

let r = Number(lines.shift())

let volume = 4 / 3.0 * PI * Math.pow(r, 3)

console.log(`VOLUME = ${volume.toFixed(3)}`)